api/clients/v2/verification/blob_verifier.go

```diff
@@ -74,3 +75,40 @@ func (v *BlobVerifier) VerifyBlobV2FromSignedBatch(
 
 	return nil
 }
+
+// VerifyBlobV2 calls the VerifyBlobV2 view function on the EigenDABlobVerifier contract
+//
+// This method returns nil if the blob is successfully verified. Otherwise, it returns an error.
+//
+// It is the responsibility of the caller to configure a timeout on the ctx, if a timeout is required.
```

Comment:
This sentence possibly not necessary, as this behavior is more or less the status-quo of a golang function that accepts a context as an argument but does not have a timeout parameter.

Reply:
Removed